src/routes/(console)/project-[project]/functions/function-[function]/createCli.svelte

```diff
@@ -127,3 +128,23 @@
         <Button secondary on:click={() => (show = false)}>Close</Button>
     </svelte:fragment>
 </Modal>
+
+<style>
+    .box {
+        padding: unset !important;
+        background-color: unset !important;
+    }
+
+    /* should we use global here? */
```

Comment:
I think we don't need a global here indeed. Does it also work without?

Reply:
It doesn't, because the supplied class is used inside the `Code` component. Lmk if there's a better alternative :)